assets/errata.js

```diff
@@ -13,115 +13,160 @@ a new comment on the issue may be added beginning with the word "Summary".
+    const p1 = document.createElement('p');
+    p1.innerHTML = `<span class='what'>Issue number:</span> <a href='${issue.html_url}'>#${issue.number}</a><br>`  +
+                    `<span class='what'>Raised by:</span> <a href='${issue.user.url}'>@${issue.user.login}</a><br>` +
+                    `<span class='what'>Extra labels:</span> ${display_labels}<br>`                                 +
+                    `<span class='what'>Pull request? </span> ${pull_request}<br>`                                  +
+                    `<span class='what'>Status:</span> <span class="${state_class}">${state}</span><br> `           ;
+    div.append(p1);
+
+    const p2 = document.createElement('p');
+    p2.innerHTML = `<span class='what'><a href='${issue.html_url}'>Initial description:</a></span> ${issue.body}`;
+    div.append(p2);
 
-$(document).ready(function() {
-    var display_issue = function(node, issue, comments, labels) {
-        var  display_labels = _.reduce(labels, function(memo, label, index) {
-            if( label === "Errata" )
-                return memo
-            else if( memo === "" )
-                return label
-            else
-                return memo + ", " + label
-        }, "");
-        var div = $("<div class='issue'></div>");
-        node.append(div);
-        div.append("<h3>\"" + issue.title + "\"</h3>");
-
-        let state = 'Open';
-        let state_class = 'state_open';
-        if (issue.state !== 'open') {
-            state = 'Closed';
-            state_class = 'state_closed';
+    // See if a summary has been added to the comment.
+    let summary = undefined;
+    comments.forEach((comment) => {
+        if( comment.body.search('^Summary:') !== -1 ) {
+            summary = comment;
         }
+    });
 
-        let pull_request = (issue.pull_request === undefined) ? "No" : "Yes";
-
-        div.append("<p><span class='what'>Issue number:</span> <a href='" + issue.html_url + "'>#" + issue.number + "</a><br>" +
-                   "<span class='what'>Raised by:</span><a href='" + issue.user.url + "'>@" + issue.user.login + "</a><br>"    +
-                   "<span class='what'>Extra labels:</span> " + display_labels + "</a><br>"                                    +
-                   `<span class='what'>Pull request?</span> ${pull_request}<br>`                                               +
-                   `<span class='what'>Status:</span> <span class="${state_class}">${state}</span><br> `                        +
-                   "</p>");
-        div.append("<p><span class='what'><a href='" + issue.html_url + "'>Initial description:</a></span> " + issue.body + "</p>");
-
-        // See if a summary has been added to the comment.
-        var summary = undefined;
-        _.each(comments, function(comment) {
-            if( comment.body.search("^Summary:") !== -1 ) {
-                summary = comment;
-            }
-        })
+    if( summary !== undefined ) {
```

Comment:
```suggestion
    if( summary ) {
```

Reply:
yep... I have got bitten by this type of shorthand in Javascript before, hence my instinctive reaction of spelling things out. But I guess in this case it is all right.